configs/mmpose/pose-detection_rtmo_tensorrt-fp16_dynamic-640x640.py

```diff
@@ -0,0 +1,36 @@
+_base_ = ['./pose-detection_static.py', '../_base_/backends/tensorrt-fp16.py']
+
+onnx_config = dict(
+    output_names=['dets', 'keypoints'],
+    dynamic_axes={
+        'input': {
+            0: 'batch',
+        },
+        'dets': {
+            0: 'batch',
+        },
+        'keypoints': {
+            0: 'batch'
+        }
+    })
+
+backend_config = dict(
+    common_config=dict(max_workspace_size=1 << 30),
+    model_inputs=[
+        dict(
+            input_shapes=dict(
+                input=dict(
+                    min_shape=[1, 3, 640, 640],
```

Comment:
if all shapes are the same, we can rename it to static config instead of dynamic